analyze_message.py

```diff
@@ -138,16 +160,18 @@ def get_command_line_args():
     """ Read command line args, of course."""
     parser = argparse.ArgumentParser()
     parser.add_argument("-n", "--no-checkbox", action="store_true")
+    parser.add_argument("-c", "--cumulative", action="store_true", help="Plot cumulative values")
     parser.add_argument("input_file", help="raw text copied from slack")
     return parser.parse_args()
 
 
-def start_server(address, port, url, attempts=100):
+def start_server(args, address, port, url, attempts=100):
+    g = partial(graph_it, args)
```

Comment:
THAT!!!!  That's what I needed there.  Awesome.  I should try to do the same for the callback rather than jamming data into the function object.

Reply:
`partial()` is one of those functions that has limited uses, but man are they powerful uses :joy: